NineChronicles.Headless/GraphTypes/StandaloneQuery.cs

```diff
@@ -120,33 +120,28 @@ blockHash switch
 
                     var recipient = context.GetArgument<Address?>("recipient");
 
-                    IEnumerable<Transaction> txs = digest.TxIds
-                        .Select(bytes => new TxId(bytes))
+                    IEnumerable<Transaction> blockTxs = digest.TxIds
+                        .Select(b => new TxId(b.ToBuilder().ToArray()))
```

Comment:
Is it needed to become `byte[]` through the array builder? I wonder its benefit.